Sources/Core/Utils/DataPersistence.swift

```diff
@@ -30,27 +30,27 @@ let kSessionFilenamePrefixV2_2 = "session"
 var sessionKey = "session"
 
 class DataPersistence: NSObject {
```

Comment:
does it still need to inherit from NSObject?

Reply:
Good point, it doesn't, changed it.